packages/orchestration/src/examples/README.md

```diff
@@ -0,0 +1,11 @@
+# Examples
+
+This directory contains sample contracts showcasing the Orchestration API. Each example demonstrates specific functionalities:
+
+- **stakeIca.contract.js**: Interchain account creation for remote staking
+- **basic-flows.contract.js**: Account creation and query sending
+- **unbondExample.contract.js**: Cross-chain unbonding and liquid staking
```

Comment:
This one is very incomplete:
 - #9782

The README could have another list of contracts that are in progress, and this one could go there.